.github/workflows/maven.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - uses: actions/setup-node@v1
+      name: Set up NodeJS v12
+      with:
+        node-version: '12'
+    - run: npm install
+      working-directory: ./ui
+    - name: Build with Maven
+      run: mvn -B package --file pom.xml
```

Comment:
will we also run tests?

Reply:
The maven tests are already run with that mvn command. You can look at the log in the "show all checks" to see it's running some JUnit tests.

When we get the jasmine tests merged we can add npm test command to this workflow